azure-pipelines/build/linux/du/templates/doclient-lite-docker-steps.yml

```diff
@@ -49,11 +49,5 @@ steps:
     SourceFolder: '/tmp/build-deliveryoptimization-agent-${{parameters.targetOsArch}}/linux-${{parameters.config}}'
     Contents: |
       client-lite/deliveryoptimization-agent.dbg
-    TargetFolder: '$(Build.ArtifactStagingDirectory)/${{parameters.targetOsArch}}-${{parameters.config}}'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/agent-${{parameters.targetOsArch}}-${{parameters.config}}'
   displayName: 'Populate debug symbols to staging dir'
-
-- task: PublishBuildArtifacts@1
```

Comment:
>PublishBuildArtifacts [](http://example.com/codeflow?start=8&length=21)

this was removed across all three. I assume the "artifact" part in the do-docker-jobs.yml does the equivalent now?

Reply:
Yes. And the same part in release-docker-build-steps.yml as well.

It was the migration tool that moved these lines from here the parent yaml, so I kept it that way.